CHANGELOG.md

```diff
@@ -90,7 +97,6 @@
         type: RenderType = "document",
         render_dependencies: bool = True,
         request: Optional[HttpRequest] = None,
-        *response_args: Any,
```

Comment:
Also, since I've already made change to the `render_to_response()` method previously, I changed it so that it accepts only addional kwargs, but not args.

For context it's that any additional args / kwargs passed to `render_to_response()` are then forwarded to the `HttpResponse` constructor.

The reason for removing the forwarding of args is that one would have to specify all other 8 (!) arguments in order to pass the additional args to `HtppResponse`.

So it makes a lot more sense to just pass the kwargs.